frontend/src/scenes/pipeline/BatchExportBackfills.tsx

```diff
@@ -0,0 +1,254 @@
+                        render: (_, backfill) => (backfill.finished_at ? <TZLabel time={backfill.finished_at} /> : ''),
+                    },
+                    {
+                        key: 'actions',
+                        width: 0,
+                        render: function RenderActions(_, backfill) {
+                            if (canEnableNewDestinations && backfillIsCancelable(backfill.status)) {
+                                return (
+                                    <div className="flex gap-1">
+                                        <BackfillCancelButton backfill={backfill} cancelBackfill={cancelBackfill} />
+                                    </div>
+                                )
+                            }
+                        },
+                    },
+                ]}
+                emptyState={
+                    <div className="space-y-2">
+                        <div>No backfills in this time range.</div>
+                        {canEnableNewDestinations && (
+                            <LemonButton type="primary" onClick={() => openBackfillModal()}>
+                                Backfill batch export
+                            </LemonButton>
+                        )}
+                    </div>
+                }
+            />
+        </>
+    )
+}
+
+function BackfillCancelButton({
+    backfill,
+    cancelBackfill,
+}: {
+    backfill: BatchExportBackfill
+    cancelBackfill: (backfill: BatchExportBackfill) => void
+}): JSX.Element {
+    return (
+        <span className="flex items-center gap-1">
+            <LemonButton
+                size="small"
+                type="secondary"
+                icon={<IconCancel />}
+                tooltip="Cancel backfill"
+                onClick={() =>
+                    LemonDialog.open({
+                        title: 'Cancel backfill?',
+                        description: (
+                            <>
+                                <p>This will cancel the selected backfill.</p>
+                            </>
+                        ),
+                        width: '20rem',
+                        primaryButton: {
+                            children: 'Cancel backfill',
+                            onClick: () => cancelBackfill(backfill),
+                        },
+                        secondaryButton: {
+                            children: 'Go back',
+                        },
+                    })
+                }
+            />
+        </span>
+    )
+}
+
+const combineFailedStatuses = (status: BatchExportBackfill['status']): BatchExportBackfill['status'] => {
+    // Eventually we should expose the difference between "Failed" and "FailedRetryable" to the user,
+    // because "Failed" tends to mean their configuration or destination is broken.
+    if (status === 'FailedRetryable') {
+        return 'Failed'
+    }
```

Comment:
You should never see `FailedRetryable` statuses bubble-up to the user though: We have no maximum attempts, so we will retry on every single retryable failure. This means that you will never see the `FailedRetryable` status in our database, because it is still `Running`.